socialhome/streams/app/components/stamped_elements/ProfileStampedElement.vue

```diff
@@ -0,0 +1,132 @@
+                    <b-dropdown-item
+                        :href="urls.accountsEmail"
+                        :title="translations.email"
+                        :aria-label="translations.email"
+                    >
+                        <i class="fa fa-envelope"></i> {{ translations.email }}
+                    </b-dropdown-item>
+                    <b-dropdown-item
+                        v-if=""
+                        :href="urls.accountsEmail"
+                        :title="translations.organizeProfileContent"
+                        :aria-label="translations.organizeProfileContent"
+                    >
+                        <i class="fa fa-arrows-v"></i> {{ translations.organizeProfileContent }}
+                    </b-dropdown-item>
+                </b-dropdown>
+            </div>
+            <div class="mt-1">
+                <b-button
+                    variant="secondary"
+                    :href="urls.contactsFollowed"
+                    :title="translations.following"
+                    :aria-label="translations.following"
+                >
+                    <i class="fa fa-user" />
+                    <i class="fa fa-arrow-right" />
+                    <i class="fa fa-users" />
+                    &nbsp;{{ profile.followingCount }}
+                </b-button>
+            </div>
+            <div class="mt-1">
+                <b-button
+                    variant="secondary"
+                    href="#"
+                    :title="translations.followers"
+                    :aria-label="translations.followers"
+                >
+                    <i class="fa fa-users" />
+                    <i class="fa fa-arrow-right" />
+                    <i class="fa fa-user" />
+                    &nbsp;{{ profile.followersCount }}
+                </b-button>
+            </div>
+        </div>
+        <div class="d-inline-block">
+            <img class="profile-stream-stamped-image" :src="profile.saferImageUrlLarge" >
+        </div>
+        <div class="d-inline-block ml-3 align-center stamped-profile-info">
+            <h1>{{ nameOrGuid }}</h1>
+            <h3><cite :title="translations.userHandle">{{ profile.handle }}</cite></h3>
+        </div>
+    </div>
+</template>
+
+<script>
+import Vue from "vue"
+import applicationStore from "streams/app/stores/applicationStore"
+
+
+export default Vue.component("profile-stamped-element", {
+    computed: {
+        nameOrGuid() {
+            return this.profile.name ? this.profile.name : this.profile.guid
+        },
+        profile() {
+            return this.$store.state.applicationStore.profile
+        },
+        showAdminButtons(){
+            return this.$store.state.applicationStore.isUserAuthenticated &&
+                this.profile.id === this.$store.state.applicationStore.currentBrowsingProfileId
+        },
+        translations() {
+            return {
+                apiToken: gettext("API token"),
```

Comment:
I wonder how do we pull out strings from our Vue files, didn't think of this before, hmm... We need to extract them into the .po files. I think it might be worth writing our own extractor 🤔 Unless Vue has something using this `computed.translations` and we can post-process that into .po files?

Reply:
I'm not sure I understand everything you wrote :)

Reply:
tld/dr; we need to rip out the strings somehow. In fact, looking at django docs, this might be supported already and just have to pass the parameter `.vue` as extra file extensions. Would have to try to know for sure.